mbq/metrics/contrib/utils.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+DIGIT_ID_REGEX = re.compile('\/[0-9]+')
+UUID_REGEX = re.compile('\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}')
+
+
+def _sluggified_path(path):
+    return re.sub(DIGIT_ID_REGEX, '/:id', re.sub(UUID_REGEX, '/:id', path))
+
+
+def compute_tags(status_code, path, method, content=None):
```

Comment:
Should probably be something like `get_response_tags`. `compute` is a bit too cute and `tags` is too generic.

Reply:
Or `get_response_metric_tags`